[2022-09-27 06:09:21,760] {processor.py:163} INFO - Started process (PID=62) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:09:21,763] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2022-09-27 06:09:21,765] {logging_mixin.py:109} INFO - [2022-09-27 06:09:21,765] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:09:21,790] {processor.py:632} INFO - DAG(s) dict_keys(['example_branch_datetime_operator_2']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:09:21,795] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:09:21,808] {logging_mixin.py:109} INFO - [2022-09-27 06:09:21,808] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:09:21,816] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:09:21,824] {logging_mixin.py:109} INFO - [2022-09-27 06:09:21,824] {dag.py:2921} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2022-09-26 00:00:00+00:00
[2022-09-27 06:09:21,857] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 0.101 seconds
[2022-09-27 06:10:01,003] {processor.py:163} INFO - Started process (PID=105) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:10:01,005] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2022-09-27 06:10:01,007] {logging_mixin.py:109} INFO - [2022-09-27 06:10:01,007] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:10:01,033] {processor.py:632} INFO - DAG(s) dict_keys(['example_branch_datetime_operator_2']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:10:01,037] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:10:01,048] {logging_mixin.py:109} INFO - [2022-09-27 06:10:01,048] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:10:01,056] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:10:01,065] {logging_mixin.py:109} INFO - [2022-09-27 06:10:01,065] {dag.py:2921} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2022-09-26 00:00:00+00:00
[2022-09-27 06:10:01,092] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 0.094 seconds
[2022-09-27 06:10:43,735] {processor.py:163} INFO - Started process (PID=153) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:10:43,737] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2022-09-27 06:10:43,738] {logging_mixin.py:109} INFO - [2022-09-27 06:10:43,738] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:10:43,763] {processor.py:632} INFO - DAG(s) dict_keys(['example_branch_datetime_operator_2']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:10:43,768] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:10:43,779] {logging_mixin.py:109} INFO - [2022-09-27 06:10:43,779] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:10:43,788] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:10:43,796] {logging_mixin.py:109} INFO - [2022-09-27 06:10:43,796] {dag.py:2921} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2022-09-26 00:00:00+00:00
[2022-09-27 06:10:43,823] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 0.093 seconds
[2022-09-27 06:11:24,791] {processor.py:163} INFO - Started process (PID=187) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:11:24,793] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2022-09-27 06:11:24,795] {logging_mixin.py:109} INFO - [2022-09-27 06:11:24,795] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:11:24,822] {processor.py:632} INFO - DAG(s) dict_keys(['example_branch_datetime_operator_2']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:11:24,827] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:11:24,838] {logging_mixin.py:109} INFO - [2022-09-27 06:11:24,838] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:11:24,846] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:11:24,853] {logging_mixin.py:109} INFO - [2022-09-27 06:11:24,853] {dag.py:2921} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2022-09-26 00:00:00+00:00
[2022-09-27 06:11:24,882] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 0.096 seconds
[2022-09-27 06:12:07,235] {processor.py:163} INFO - Started process (PID=224) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:12:07,237] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2022-09-27 06:12:07,239] {logging_mixin.py:109} INFO - [2022-09-27 06:12:07,239] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:12:07,267] {processor.py:632} INFO - DAG(s) dict_keys(['example_branch_datetime_operator_2']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2022-09-27 06:12:07,272] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:12:07,283] {logging_mixin.py:109} INFO - [2022-09-27 06:12:07,283] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:12:07,291] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:12:07,299] {logging_mixin.py:109} INFO - [2022-09-27 06:12:07,299] {dag.py:2921} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2022-09-26 00:00:00+00:00
[2022-09-27 06:12:07,328] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 0.098 seconds
